pkg/monitor/scheduledevent/scheduled-event-monitor_internal_test.go

```diff
@@ -91,28 +79,34 @@ func TestUncordonAfterRebootPreDrainSuccess(t *testing.T) {
 }
 
 func TestUncordonAfterRebootPreDrainMarkWithEventIDFailure(t *testing.T) {
-	resetFlagsForTest()
-
 	tNode := getNode(t, getDrainHelper(fake.NewSimpleClientset()))
 	err := uncordonAfterRebootPreDrain(monitor.InterruptionEvent{}, *tNode)
 	h.Assert(t, err != nil, "Failed to return error on MarkWithEventID failing to fetch node")
 }
 
 func TestUncordonAfterRebootPreDrainNodeAlreadyMarkedSuccess(t *testing.T) {
-	resetFlagsForTest()
+	nthConfig := config.Config{
+		DryRun:   true,
+		NodeName: nodeName,
+	}
 
 	client := fake.NewSimpleClientset()
 	//nolint:errcheck
-	client.CoreV1().Nodes().Create(&v1.Node{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: nodeName,
-		},
-		Spec: v1.NodeSpec{
-			Unschedulable: true,
+	_, err := client.CoreV1().Nodes().Create(context.Background(),
```

Comment:
shouldn't the `//nolint:errcheck` let us ignore `err` here? There's times where you're catching the `err` value and other times you ignore it, I think we should make it consistent one way or the other

Reply:
yeah I think I prefer to just check the errs in all cases since it's the same number of lines, will update.